bubble-wrap.yaml

```diff
@@ -167,8 +167,8 @@ global:
     sdk_shield_color: [0.506,0.192,0.169]
     sdk_shield_text_color: white
     #
-    interactive_dev: false
-    interactive_prod: true
+    # enable interactivity for key features
+    interactive: true
```

Comment:
This could also be `ux_interactive` or `sdk_interactive` (what is naming criteria for ux vs. sdk?) for namespacing and consistency.